frontend/src/styles/base/_colours.scss

```diff
@@ -1,5 +1,6 @@
 // Colours
 $white: #ffffff;
+$transparent-white: rgba(255, 255, 255, 0.2);
```

Comment:
Was thinking for colors with transparency, we can standardise and use `adjust-color($white, $alpha: -0.8)` instead of creating a new variable. A couple of things this will help with are: 
1. This keeps the variables aligned with the design system
2. For other colors, it's easier to remain consistent since we can reuse the variables instead of having to change to rgba values. E.g. `adjust-color($primary-light, $alpha: -0.7)`

One thing I observed while refactoring the colors out was that there could be multiple variants of "transparent-blue", each with a different alpha value. What do you think?

Reply:
I agree, I was not aware of this sass function! I think this definitely prevents the spawn of multiple `$transparent` valued variables and we can stick to a pre-defined set of colour variables. 

However so far our usage of `adjust-color(color, change in alpha)` has been limited to the alpha param, which is equivalent to using `rgba(color, alpha)`. Personally I find using `rgba()` easier to read and understand as it uses the desired alpha value vs using `adjust-color` which we have back-calculate the alpha value and define the difference. Let me know if you feel otherwise?

Reply:
Thats true. And given that it's quite easy to get the rgb values from Figma anyway. This is fine!